components/feature/qr/src/test/java/mozilla/ext/TestCallback.kt

```diff
@@ -0,0 +1,14 @@
+package mozilla.ext
```

Comment:
nit: license

Reply:
Yep!

Reply:
It'll be nice to add lint rule for this in the future. :)